applications/nrf5340_audio/README.rst

```diff
@@ -782,6 +782,12 @@ Selecting the CIS bidirectional communication
 The CIS unidirectional mode is the default operating mode for the application.
 You can switch to the bidirectional mode by adding the :kconfig:option:`CONFIG_STREAM_BIDIRECTIONAL` Kconfig option set to ``y``  to the :file:`prj.conf` file (for the debug version) or to the :file:`prj_release.conf` file (for the release version).
 
+Enabling the walkie talkie demo
+-------------------------------
+
+The walkie talkie demo is a bidirectional stream using the PDM microphone as input on each side.
+You can switch to the walkie talkie by adding the :kconfig:option:`CONFIG_WALKIE_TALKIE_DEMO` Kconfig option set to ``y``  to the :file:`prj.conf` file (for the debug version) or to the :file:`prj_release.conf` file (for the release version).
+
 .. note::
    Only one headset can be connected when using the bidirectional mode.
```

Comment:
Move the note to line 784.

Reply:
It is true for both modes, so I will instead do: 
```suggestion
.. note::
   Only one headset can be connected when using the bidirectional mode or walkie-talkie demo.
```